doc/changelogs/CHANGELOG_V12.md

```diff
@@ -73,6 +74,35 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="12.22.4"></a>
+## 2021-07-27, Version 12.22.4 'Erbium' (LTS), @richardlau
+
+### Notable Changes
+
+tbd
+
```

Comment:
I guess this can be removed if there's nothing of note

Reply:
There's nothing of note... yet. I've been asked to add a few more commits to this, which will be notable, but they're not ready yet.